src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -370,9 +387,10 @@ protected final RestChannelConsumer prepareRequest(RestRequest request, NodeClie
 		// check if request is authorized
 		String authError = restApiPrivilegesEvaluator.checkAccessPermissions(request, getEndpoint());
 
+		final User user = (User) threadPool.getThreadContext().getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
+		final String userName = user == null ? null : user.getName();
```

Comment:
Are these changes supposed to be backported? The original PR had more changes

Reply:
Hi Dinu, These seem to be related to the following commits - https://github.com/opendistro-for-elasticsearch/security/commit/67fe5d2b28b28d28987830d4aa734c113c3425cd
and
https://github.com/opendistro-for-elasticsearch/security/commit/ff00f4f98cf3bab9d90f56e7d6ea7ba9e796eda3.

I was backporting all commits for my changes for rolesmapping and Palash's changes for https://github.com/opendistro-for-elasticsearch/security/commit/e97d30d204c42c1a9518c85039144fcdadc0e590
and
https://github.com/opendistro-for-elasticsearch/security/commit/72ce765f0a767d609ad392c35bd8e72f3c03dd41. 

These changes are only related to the rolesmapping and saml related fixes. Let me know if while backporting we should backport other Audit log/whitelisting related changes as well.